src/nativefs/file-save-nativefs.mjs

```diff
@@ -16,15 +16,10 @@
 // @license © 2020 Google LLC. Licensed under the Apache License, Version 2.0.
 
 /**
- * Saves a file to disk using the Native File System API.
- * @param {Blob} blob - To-be-saved blob.
- * @param {Object} [options] - Optional options object.
- * @param {string} options.fileName - Suggested file name.
- * @param {string} options.description - Suggested file description.
- * @param {string[]} options.extensions - Suggested file extensions.
- * @param {FileSystemHandle} [handle] - Optional file handle to save in place.
+ * Saves file to disk using the (legacy) Native File System API.
+ * @type { typeof import("../../index").fileSave }
  */
-export default async (blob, options = {}, handle = null) => {
+export default async function fileSave (blob, options = {}, handle = null) {
```

Comment:
Can we not do this here, or do it everywhere else, too? I have no strong opinion either way, just would like consistency.

Reply:
oops, that's a leftover from trying to make TS work :D. Will remove.